src/components/data_comps/dlnd/cime_config/config_component.xml

```diff
@@ -44,6 +44,28 @@
     set to zero and not utilized. The default is NULL.</desc>
   </entry>
 
+  <entry id="DLND_CPLHIST_DOMAIN_FILE">
+    <type>char</type>
+    <valid_values></valid_values>
+    <default_value>null</default_value>
+    <values match="last">
+      <value compset="2000.*_DLND%SCPL" grid="l%0.9x1.25">$LND_DOMAIN_PATH/$LND_DOMAIN_FILE</value>
```

Comment:
Why are there these duplicated entries for all the different time periods? Why not have a single entry with a compset match of "_DLND%SCPL"?

Reply:
Another good catch! You're right. I'll change that.